Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBar.xaml

```diff
@@ -0,0 +1,299 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="ms-appx:///Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBarItem.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+
+    <DataTemplate x:Key="NormalTabTemplate">
+        <NavigationViewItem Content="{Binding Header}" />
+    </DataTemplate>
+    <DataTemplate x:Key="ContextualTabTemplate">
+        <NavigationViewItem Content="{Binding Header}">
+            <NavigationViewItem.Resources>
+                <SolidColorBrush x:Key="TopNavigationViewItemForeground" Color="{ThemeResource SystemAccentColor}" />
```

Comment:
Is this just adjusting the text? Looking at office looks like the new pattern is to just color the text:

Selected
![image](https://user-images.githubusercontent.com/24302614/106988122-cb76aa00-6723-11eb-97f6-a2831514815d.png)

vs Unselected
![image](https://user-images.githubusercontent.com/24302614/106988138-d598a880-6723-11eb-8479-1ad96edd1949.png)

And Office Online adds a bit of a different background shade to them:
![image](https://user-images.githubusercontent.com/24302614/106988229-1690bd00-6724-11eb-93d5-5f4fc374b319.png)

(Which is probably what we may want to do too.)

I like the idea of using the AccentColor as the default, but we should expose these as named resources within our dictionary set to these values so they're more easily overridable. [TabView had examples of this](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/rel/6.1.1/Microsoft.Toolkit.Uwp.UI.Controls/TabView/TabView.xaml).

Reply:
![image](https://user-images.githubusercontent.com/7549325/106990538-0e437c80-673a-11eb-9a47-c5a1ce71dbbc.png)
Yes, right now it only adjusts the text foreground. I'll set up more default resources and look into changing the hover/selected background colors.

Reply:
Thanks Yoshi, it's looking really slick. I think the main thing would be if we just have a slightly lighter tone for the background of the contextual item, you can see above it's like that in the web world even if it's not selected (it's the same if it's selected to). This is a hover example:

![image](https://user-images.githubusercontent.com/24302614/107003280-5e720d00-6741-11eb-9fad-e2cdc121aacc.png)

And if we name them as resources, folks can override them if they'd like.

I think the online versions are more akin to this pattern and more up-to-date with the direction of styling, so it'd be cool to mimic that a bit more with having the toned background to make the contextual tab stand-out a bit more beyond the accent color.

This is a real nitpicky detail, so having the Visibility binding working is the main priority (see my updated sample example I pasted in the other comment). If we could have that toggleswitch just have the Contextual tab appear with the binding all in XAML, I think we're golden. 🙂

Reply:
The Office ribbon uses Semibold for selected tabs, which may be important to include, if colour is used for contextual tabs, which wont work so well with a lower opacity for unselected tab headers.

Reply:
> The Office ribbon uses Semibold for selected tabs, which may be important to include, if colour is used for contextual tabs, which wont work so well with a lower opacity for unselected tab headers.

I did consider adding that, but I couldn't figure out how. At the moment, tab headers aren't required to be text, so I can't think of a clean way to make text bold without reaching into the visual tree using code behind. I'm sure it's possible, but I'm not experienced enough with XAML to do it.

Reply:
> Thanks Yoshi, it's looking really slick. I think the main thing would be if we just have a slightly lighter tone for the background of the contextual item, you can see above it's like that in the web world even if it's not selected (it's the same if it's selected to). This is a hover example:
> 
> ![image](https://user-images.githubusercontent.com/24302614/107003280-5e720d00-6741-11eb-9fad-e2cdc121aacc.png)
> 
> And if we name them as resources, folks can override them if they'd like.

Here's what I've done so far. Unfortunately, Office Online doesn't have an official dark mode, so I've just picked the UWP brush resource that matched in light mode and looked plausible in dark mode.
![image](https://user-images.githubusercontent.com/7549325/107163356-b93c7c00-696e-11eb-9610-94eec218f2eb.png)
![image](https://user-images.githubusercontent.com/7549325/107163377-c6596b00-696e-11eb-9ccb-492d644dea6c.png)